floyd/cli/experiment.py

```diff
@@ -164,26 +165,27 @@ def delete(ids, yes):
     """
     Delete project runs
     """
-    
+    failures = False
     for id in ids:
         experiment = ExperimentClient().get(id)
-        task_instance_id = get_module_task_instance_id(experiment.task_instances)
-        task_instance = TaskInstanceClient().get(task_instance_id) if task_instance_id else None
-
+        if not experiment:
+            failures = True
+            continue
 
-        if experiment.state in ["queued", "running"]:
-            floyd_logger.info("Experiment {}: In {} state and cannot be deleted. Stop it first".format(experiment.name, experiment.state))
+        if not yes and not click.confirm("Delete Run: {}?".format(experiment.name),
+                                         abort=False,
+                                         default=False):
+            floyd_logger.info("Experiment {}: Skipped.".format(experiment.name))
             continue
 
-        if not yes:
-            if not click.confirm("Delete Run: {}?".format(experiment.name), abort=False, default=False):
-                floyd_logger.info("Experiment {}: Skipped.".format(experiment.name))
-                continue
+        task_instance_id = get_module_task_instance_id(experiment.task_instances)
+        task_instance = TaskInstanceClient().get(task_instance_id) if task_instance_id else None
 
         if task_instance and task_instance.module_id:
             ModuleClient().delete(task_instance.module_id)
```

Comment:
Yeah I think adding this to the failures is a good idea. However, the cli currently doesn't offer a way to delete a module directly by its id, so we may wish to implement that later depending on how we resolve all the comments in this PR.